interpreter/src/main/java/org/qbicc/interpreter/impl/VmImpl.java

```diff
@@ -1214,6 +1215,15 @@ public static VmImpl create(CompilationContext ctxt, Consumer<VmObject> manualIn
         return new VmImpl(Assert.checkNotNullParam("ctxt", ctxt), manualInitializer);
     }
 
+    public VmImpl setPropertyDefines(List<String> propDefines) {
+        this.propertyDefines = propDefines;
+        return this;
+    }
+
+    List<String> getPropertyDefines() {
+        return propertyDefines;
+    }
+
```

Comment:
I don't really like this way of passing information from the front end/configuration to the back end... but until we have #991 worked out this will do. Once we have some kind of "proper" configuration relay that the run time can consume, we'll want to revisit this.